entry/src/main/java/com/example/appcrashtracker/slice/MainAbilitySlice.java

```diff
@@ -0,0 +1,39 @@
+package com.example.appcrashtracker.slice;
+
+import ohos.aafwk.ability.AbilitySlice;
+import ohos.aafwk.content.Intent;
+import ohos.agp.components.Button;
+import ohos.hiviewdfx.HiLog;
+import ohos.hiviewdfx.HiLogLabel;
+import com.example.appcrashtracker.ResourceTable;
+
+/** This is the main ability slice. */
+public class MainAbilitySlice extends AbilitySlice {
+    private static final String CONST = "Exception";
+
+    @Override
+    public void onStart(Intent intent) {
+        super.onStart(intent);
+        super.setUIContent(ResourceTable.Layout_ability_main);
+        Button bt = (Button) findComponentById(ResourceTable.Id_crashButton);
+
+        bt.setClickedListener(component -> crashme());
+    }
+
+    private void crashme() {
```

Comment:
Please follow the camel case convention in naming.

Reply:
Resolved in commit 00bd46e77851473750360a5a91d148105bf9950c